tests/snuba/api/endpoints/test_discover_key_transactions.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import absolute_import
+
+import six
+
+from django.core.urlresolvers import reverse
+
+from sentry.discover.models import KeyTransaction, MAX_KEY_TRANSACTIONS
+from sentry.utils.samples import load_data
+from sentry.testutils import APITestCase
+
+
+class KeyTransactionTest(APITestCase):
+    def setUp(self):
+        super(KeyTransactionTest, self).setUp()
+
+        self.login_as(user=self.user, superuser=False)
+
+        self.org = self.create_organization(owner=self.user, name="foo")
+
+        self.project = self.create_project(name="bar", organization=self.org)
+
+    def test_save_key_transaction(self):
```

Comment:
Would be good to have tests for projects you don't have access to.

Reply:
Should be covered by `test_save_with_wrong_project` :+1: but also going to add a test for deletes